frontend/components/equipment-card.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <v-card border="md" class="d-flex flex-row" @click="console.log()">
+    <v-card-text class="border-e-md d-flex flex-column justify-space-between align-center">
+      <h2 class="text-h6 text-center">
+        {{ equipment.name }}
+      </h2>
+      <v-progress-circular
+        :model-value="equipment.oee"
+        size="150"
+        width="20"
+        :color="getColorState(equipment.oee)"
+        class="text-h6 mt-2">
+        <span>
+          <span :style="{fontSize: '1rem'}" class="font-weight-medium">OEE</span>
+          <br>
+          {{Math.round(equipment.oee)}}%
+        </span>
+      </v-progress-circular>
+    </v-card-text>
+
+    <v-card-text class="pa-0">
+      <v-card
+        v-for="(metric, metricIndex) in metrics"
+        :key="metric.label"
+        class="pa-0 d-flex flex-column justify-center"
+        :class="metricIndex +1 === metrics.length ? '' : 'border-b-md'"
+        height="33%"
```

Comment:
Instead of setting a specific height, which wouldn't respond well to changing number of metrics (let alone the extra 1% we're not taking up), make the parent element a `flex-col` and make sure every child element has a flex basis of 0 (e.g. `flex: 1 1 0px`. This will make all items the same height.
Or, if we specifically want to fill 3 slots, use a grid layout with 3 rows, `1fr` each!

Reply:
[26f67f4](https://github.com/cesmii/OEE-SMIP-App/commit/26f67f48ea82f2d58e911d339f2af6a90fe4a41a)